src/pages/index.js

```diff
@@ -174,7 +173,7 @@ function createCard(cardData) {
       handleImageClick: (data) => {
         modalWithImage.open(data);
       },
-      handleDeleteClick: () => {
+      deleteCard: () => {
```

Comment:
This method can be called `handleDeleteClick`, you should rename the method, which removes the card element, so the name will describe the action `handleDeleteIcon` => `deleteCard`